### ISSUE TITLE
Apktool not working with latest build tools: AndroidManifest.xml:3: error: No resource identifier found for attribute 'appComponentFactory' in package 'android'

### ISSUE THREAD

**Issue by mandshashi** [iBotPeaches/Apktool#1867]
### Information
1. **Apktool Version (`apktool -version`)** - 2.3.3
2. **Operating System (Mac, Linux, Windows)** - Mac
3. **APK From? (Playstore, ROM, Other)** - Other

### Stacktrace/Logcat
```
AndroidManifest.xml:3: error: No resource identifier found for attribute 'appComponentFactory' in package 'android'
at com.fiberlink.android.wrapping.Main.main(Main.java:60)
Exception in thread "main" brut.androlib.AndrolibException: brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/Users/mac/aapt, p, --min-sdk-version, 22, --target-sdk-version, 25, --version-code, 1, --version-name, 1.0, --no-version-vectors, -F, /var/folders/c8/ch9xlkcs3f1cpx6z132x82x80000gn/T/APKTOOL8128108087309233550.tmp, -0, arsc, -0, META-INF/android.arch.core_runtime.version, -0, META-INF/android.arch.lifecycle_livedata-core.version, -0, META-INF/android.arch.lifecycle_runtime.version, -0, META-INF/android.arch.lifecycle_viewmodel.version, -0, META-INF/androidx.appcompat_appcompat.version, -0, META-INF/androidx.asynclayoutinflater_asynclayoutinflater.version, -0, META-INF/androidx.coordinatorlayout_coordinatorlayout.version, -0, META-INF/androidx.core_core.version, -0, META-INF/androidx.cursoradapter_cursoradapter.version, -0, META-INF/androidx.customview_customview.version, -0, META-INF/androidx.documentfile_documentfile.version, -0, META-INF/androidx.drawerlayout_drawerlayout.version, -0, META-INF/androidx.fragment_fragment.version, -0, META-INF/androidx.interpolator_interpolator.version, -0, META-INF/androidx.legacy_legacy-support-core-ui.version, -0, META-INF/androidx.legacy_legacy-support-core-utils.version, -0, META-INF/androidx.loader_loader.version, -0, META-INF/androidx.localbroadcastmanager_localbroadcastmanager.version, -0, META-INF/androidx.print_print.version, -0, META-INF/androidx.slidingpanelayout_slidingpanelayout.version, -0, META-INF/androidx.swiperefreshlayout_swiperefreshlayout.version, -0, META-INF/androidx.vectordrawable_vectordrawable-animated.version, -0, META-INF/androidx.vectordrawable_vectordrawable.version, -0, META-INF/androidx.versionedparcelable_versionedparcelable.version, -0, META-INF/androidx.viewpager_viewpager.version, -0, arsc, -I, /var/folders/c8/ch9xlkcs3f1cpx6z132x82x80000gn/T/1.apk, -S, 
```

### Steps to Reproduce
1. `apktool `

### Frameworks
If this APK is from an OEM ROM (Samsung, HTC, LG). Please attach framework files
(`.apks` that live in `/system/framework` or `/system/priv-app`)

### APK
If this APK can be freely shared, please upload/attach a link to it.

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything?
2. If you are trying to install a modified apk, did you resign it?
3. Are you using the latest apktool version?

**Comment by iBotPeaches** [comment]
Do you have the application in question? Looks like this might be a new resource in base framework, but difficult to determine without app.

**Comment by mandshashi** [comment]
@iBotPeaches Created a sample app and attached. Please have a look. Getting below exception with the sample app.
"error: No resource identifier found for attribute 'appComponentFactory' in package 'android'"
[app-debug.apk.zip](https://github.com/iBotPeaches/Apktool/files/2301550/app-debug.apk.zip)

**Comment by iBotPeaches** [comment]
```
➜  1867 apktool d app-debug.apk 
I: Using Apktool 2.4.2-f545c2-SNAPSHOT on app-debug.apk
I: Loading resource table...
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: /home/ibotpeaches/.local/share/apktool/framework/1.apk
I: Regular manifest package...
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Baksmaling classes.dex...
I: Copying assets and libs...
I: Copying unknown files...
I: Copying original files...
➜  1867 apktool b app-debug
I: Using Apktool 2.4.2-f545c2-SNAPSHOT
I: Checking whether sources has changed...
I: Smaling smali folder into classes.dex...
I: Checking whether resources has changed...
I: Building resources...
I: Building apk file...
I: Copying unknown files/dir...
I: Built apk...
➜  1867 
```

Thanks for apk. Looks resolved on upcoming v2.5.0 release (out by Christmas). Marking as done and closing.
